### PR TITLE
Add gpu info

### PR DESCRIPTION
**Pull request by gaoteng-git**
1. Add GPU device info.
2. Pass title as “GPU Summary” to UI.
3. Change "------" to "`<hr/>`"

Corresponding front-end PR: https://github.com/pytorch/kineto/pull/264

image

Note: The 1st and 2nd V100 have different memory size, interesting....
image
image

### CODE REVIEW

File: tb_plugin/torch_tb_profiler/run.py [hunk 1]
```diff
@@ -124,19 +125,50 @@ def append_gpu_metrics(self, raw_data):
         return raw_data
 
 
+    def get_gpu_info(self, gpu_id):
```
Comment (guotuofeng) [inline tb_plugin/torch_tb_profiler/run.py:128]: could we reuse the _generate_gpu_info in run_generator.py?

Comment (guotuofeng) [reply tb_plugin/torch_tb_profiler/run.py:128]: since most of the code same with that function

Comment (gaoteng-git) [reply tb_plugin/torch_tb_profiler/run.py:128]: Reasonable. Should reuse. Done. Thanks!

File: tb_plugin/torch_tb_profiler/run.py [hunk 2]
```diff
@@ -130,13 +131,22 @@ def get_gpu_metrics_data(profile):
...
             for gpu_id in profile.gpu_ids:
                 if not is_first:
                     # Append separator line for beautiful to see.
-                    gpu_metrics_data.append({"title": "--------",
+                    gpu_metrics_data.append({"title": "<hr/>",
                                              "value": ""})
+
                 gpu_metrics_data.append({"title": "GPU {}:".format(gpu_id),
                                          "value": ""})
+                gpu_info = profile.gpu_infos.get(gpu_id, None)
```
Comment (guotuofeng) [inline tb_plugin/torch_tb_profiler/run.py:143]: need we check the gpu_infos be null or not?

**Review by guotuofeng — APPROVED** [review]

File: tb_plugin/torch_tb_profiler/run.py (hunk 2 above)
Comment (gaoteng-git) [reply tb_plugin/torch_tb_profiler/run.py:143]: Don't need in this case. Initialized as {} in run_generator.py

**Pull request merged**